core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -228,13 +230,21 @@ protected void doStop() throws Exception {
 
     List<URI> listeners = parseListeners(config.getList(RestConfig.LISTENERS_CONFIG),
             config.getInt(RestConfig.PORT_CONFIG), Arrays.asList("http", "https"), "http");
+
+    // Make an HttpConfiguration that turns off the server version header
```

Comment:
nit: remove comment - it's unnecessary (and these things tend to get out of date).

Reply:
Done